main.tf

```diff
@@ -0,0 +1,37 @@
+# Using tf_ansible module
+
+resource "aws_instance" "default" {
```

Comment:
This must be launched in a VPC. This should also define a security group. Please look at `tf_bastion`